api/src/app.py

```diff
@@ -67,6 +68,8 @@ def create_app() -> APIFlask:
     if os.getenv("ENVIRONMENT") == "local":
         initialize_jwt_auth()
 
+    init_newrelic()
```

Comment:
Doesn't this need to happen before anything else? We did that before / the New Relic docs say that needs to happen.

https://docs.newrelic.com/install/python/?python-non-web=non-web-yes#integrating

---

It might be that this needs to be done for multi-threading purposes, which we only do non-locally unless you reconfigure some commands.

You'll need to swap out the command temporarily at https://github.com/HHS/simpler-grants-gov/blob/main/api/docker-compose.yml#L78 for what we use in prod to run via Gunicorn: https://github.com/HHS/simpler-grants-gov/blob/main/api/Dockerfile#L127

Reply:
@chouinar What if we just lift init_newrelic() out into app.py, like I did here? Would we still need to change the Docker command?

Reply:
I'm suggesting you adjust the docker command to test that it works in multi-threaded/multi-process mode, not as a permanent change